packages/inventory-compliance/src/SystemPolicyCard.js

```diff
@@ -63,77 +55,70 @@ class SystemPolicyCard extends React.Component {
-                                    'The system compliance score is calculated by OpenSCAP and ' +
-                                    'is a normalized weighted sum of rules selected for this policy.'
-                                }>
-                                <span>(Score: { passedPercentage })</span>
-                            </Tooltip>
-                        </Text>
-                    </div>
-                    <div className='margin-bottom-md' >
+        return <Card>
+            <CardBody>
+                <TextContent className='margin-bottom-md'>
+                    <Text className='margin-bottom-top-none'
+                        component={ TextVariants.h4 }
+                        onMouseEnter={ this.onTitleMouseover }
+                        onMouseLeave={ this.onTitleMouseout }
+                    >
+                        { cardTitle }
+                    </Text>
+                    { policy ?
                         <Text
-                            component={ TextVariants.medium }
-                            onMouseEnter={ this.onMouseover }
-                            onMouseLeave={ this.onMouseout }
-                            className='wrap-break-word'
-                        >
-                            Profile: { refIdTruncated }
-                            <br/>
-                            SSG version: { benchmark.version }
+                            style={{ color: 'var(--pf-global--Color--200)' }}
+                            component={ TextVariants.small }
+                            onMouseEnter={ this.onSubTitleMouseover }
+                            onMouseLeave={ this.onSubTitleMouseout }>
+                            { cardSubTitle }
                         </Text>
-                    </div>
-                    <Text className='margin-bottom-none' component={ TextVariants.small }>
-                      Last scanned: <FormattedRelativeCmp value={ Date.parse(lastScanned) } />
+                        :
+                        <Tooltip position='bottom' content={
+                            <TextContent>
+                                This policy report was uploaded into the Compliance application.
+                                If you would like to manage your policy inside the Compliance application,
+                                use the &quot;Create a policy&quot; wizard to create one and associate systems.
+                            </TextContent>
+                        }>
+                            External policy <OutlinedQuestionCircleIcon className='grey-icon'/>
+                        </Tooltip>
+                    }
+                </TextContent>
+                { supported && <div className='margin-bottom-md' >
+                    { this.complianceIcon(compliant) }
+                    <Text component={ TextVariants.small }>
+                        { rulesFailed } rule{ rulesFailed === 1 ? '' : 's' } failed
```

Comment:
Number of failed rules should be shown also for unsupported, as per mocks.

Reply:
They are back in now. Only the compliance icon above will be hidden.